File: library/src/main/java/com/github/jokar/multilanguages/library/LanguageLocalListener.java
package com.github.jokar.multilanguages.library;

import android.content.Context;

import java.util.Locale;

/**
 * Create by JokAr. on 2019-07-08.
 */
public interface LanguageLocalListener {

    /**
     * 获取选择设置语言
     *
     * @param context
     * @return
     */
    Locale getSetLanguageLocale(Context context);
}

File: library/src/main/java/com/github/jokar/multilanguages/library/LanguageLocalListener.kt
package com.github.jokar.multilanguages.library

import android.content.Context
import java.util.*

/**
 * Create by JokAr. on 2019-07-08.
 */
interface LanguageLocalListener {
    /**
     * 获取选择设置语言
     *
     * @param context
     * @return
     */
    fun getSetLanguageLocale(context: Context?): Locale?
}